demo/demo.js

```diff
@@ -178,9 +178,10 @@ function storeParams(formData) {
  * @returns {ParsedRevUrl}
  */
 export function parseRevUrl(url) {
+	url = url.trim();
```

Comment:
One note - in this demo page we can assume URL is a string, but in other contexts this could be passed a `URL` instead or undefined instead of a string, hence wrapping in `${url}`

Reply:
This url is a string, not a URL object. In ts I would add a type annotation, but calling with a non-string is not something to deal with inside this method.